bundles/org.openhab.binding.dwdpollenflug/src/main/java/org/openhab/binding/dwdpollenflug/internal/DWDPollenflugBindingConstants.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.dwdpollenflug.internal;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+
+/**
+ * The {@link DWDPollenflugBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Johannes DerOetzi Ott - Initial contribution
+ */
+@NonNullByDefault
+public class DWDPollenflugBindingConstants {
+    public static final String DATE_PATTERN = "yyyy-MM-dd HH:mm";
+
+    public static final String EMPTY = "";

```

Comment:
Please do not use a constant for an empty string. Just use `""` where needed.

Reply:
Removed